### DISCUSSION TITLE
42 errors

### DISCUSSION THREAD

**Discussion by dracojoaquin1** [enitimeago/make-it-mmd#80] [Q&A] [answered]
i have a lot of erros i try using 1.1.1 and the 1.2.0 beta 1 is always give me all erros and not work any idea?
![image](https://github.com/user-attachments/assets/182d0a4f-a5b5-485b-b221-1d3b2e78592c)

**Comment by dracojoaquin1** [comment, marked as answer]
using the VCC works 
![image](https://github.com/user-attachments/assets/7700d837-e0db-4f04-af30-42ee4716a51e)
using .unity packages not work
![image](https://github.com/user-attachments/assets/beb112b8-7782-4a22-b6c9-3e150ebfb028)

**Reply by enitimeago** [reply]
In order for Make It MMD to work, [Non-Destructive Modular Framework](https://github.com/bdunderscore/ndmf) must also be installed in your project.

It seems like it worked for you once you used VCC, because installing from VCC will automatically install NDMF for you as well.

Thanks for raising this, I'll improve the documentation to make this clearer.

**Comment by dracojoaquin1** [comment]
also i have a question can make it in english because i literaly not understand how using it even with the tutorials where say this things![image](https://github.com/user-attachments/assets/86485dd6-e44b-4fa0-8bdc-11608fdc5bcf) i think is better have a option to make in english for people know what to put there and not using blender u..u

**Reply by enitimeago** [reply]
Thanks for the feature request. It definitely would be good if the MMD names can be displayed in English.

I recommend using this chart right now to decide what to put where. Even if the names were displayed in English, it's not obvious what they mean.

https://www.deviantart.com/xoriu/art/MMD-Facial-Expressions-Chart-341504917
